src/Pages/Ootd/Main/Modal.js

```diff
@@ -0,0 +1,185 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable jsx-a11y/alt-text */
+import React, { Component } from 'react';
+import Comment from './Comment';
+import '../../../Styles/common.scss';
+import '../../../Styles/reset.scss';
+import './Modal.scss';
+
+class Modal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      followBtn: false,
+      comment: "",
+      comments:[],
+      description: this.props.modalData?.modalData?.modalData?.description,
+      res: [],
+      value: "",
+    }
+  }
+  num = 0;
+
+  followBtnActive = () => {
+    if(this.state.followBtn) {
+      this.setState({
+        followBtn: false
+      })
+    } else {
+      this.setState({
+        followBtn: true
+      })
+    }
+    }
+
+  getData = (e) => {
+  e.preventDefault();
+
+    this.setState({
+      comment: e.target.value,
+    })
+    this.props.getData({
+      getData: e.target.value,
+    })
+  }
+
+  API = `http://192.168.219.101:8000/ootds/1/comments`;
+
+  handleKeyPress = (e) => {
+    e.preventDefault();
+    if(e.key === "Enter") {
+      if(!this.state.comment) {
+        e.preventDefault();
+      } else {
+        this.setState({value: e.target.value})
+        this.handleComment();
+      }
+    }
+  }
+
+  handleComment = (e) => {
+    e.preventDefault();
+    fetch(this.API, {
+      method: "POST",
+      body: JSON.stringify({
+        content: this.state.value,
+        user_id: this.props.modalData?.modalData?.modalData?.id,
+      })
+    }).then((res) => res.json())
+    .then((res) => 
+      this.setState({
+        res: res,
+      })
+    )
+    const { comment, comments } = this.state;
+    comments.push({comment});
+      this.setState({
+      comment: ""
+    });
+    this.num += 1;
+  }
+
+  render() {
+    console.log(this.props.modalData?.modalData?.modalData?.id);
+    const { closeModal, id, follower, handleClickLike, modalData } = this.props;
+    const { followBtn, comments } = this.state;
+    
+  return (
+    <>
+      <div className="modalWrapper" onClick={closeModal}>
+        <div className="innerWrapper" onClick={(e) => e.stopPropagation()}>
+          <div className="leftWrapper">
+            <div className="imgWrapper">
+              <img className="image" src={modalData?.modalData?.modalData?.contentImg} />
+              <div className={modalData?.modalData?.modalData?.contentImg?.length > 1 ? "smallImages" : "displayNone"}>
+                <figure className="smallImagesWrapper"><img className="smallImage" src={modalData?.modalData?.modalData?.contentImg[0]} /></figure>
+                <figure className="smallImagesWrapper"><img className="smallImage" src={modalData?.modalData?.modalData?.contentImg[1]} /></figure>
```

Comment:
줄 띄어주기 잘 해주세요..! 
+ map사용 해도 좋을 코드처럼 보이네요!
```suggestion
                <figure className="smallImagesWrapper">
                  <img className="smallImage" src={modalData?.modalData?.modalData?.contentImg[0]} />
                </figure>
                <figure className="smallImagesWrapper">
                  <img className="smallImage" src={modalData?.modalData?.modalData?.contentImg[1]} />
                </figure>
```

Reply:
맞습니다!! map 사용해서 댓글 데이터를 백엔드에서 받아쓰는 방향으로 하겠습니다!
위 코멘트와 겹쳐지는 부분인 것 같아요!